content/references/http-websocket-apis/public-api-methods/ledger-methods/ledger-clio.md

```diff
@@ -0,0 +1,576 @@
+| `retries_remaining` | Number           | How many times this transaction can be retried before being dropped. |
+| `preflight_result`  | String           | The tentative result from preliminary transaction checking. This is always `tesSUCCESS`. |
+| `last_result`       | String           | _(May be omitted)_ If this transaction was left in the queue after getting a [retriable (`ter`) result](ter-codes.html), this is the exact `ter` result code it got. |
+| `auth_change`       | Boolean          | _(May be omitted)_ Whether this transaction changes this address's [ways of authorizing transactions](transaction-basics.html#authorizing-transactions). |
+| `fee`               | String           | _(May be omitted)_ The [Transaction Cost](transaction-cost.html) of this transaction, in [drops of XRP][]. |
+| `fee_level`         | String           | _(May be omitted)_ The transaction cost of this transaction, relative to the minimum cost for this type of transaction, in [fee levels][]. |
+| `max_spend_drops`   | String           | _(May be omitted)_ The maximum amount of [XRP, in drops][], this transaction could potentially send or destroy. |
+
+If the request specified `"owner_funds": true` and expanded transactions, the response has a field `owner_funds` in the `metaData` object of each [OfferCreate transaction][]. The purpose of this field is to make it easier to track the [funding status of offers](offers.html#lifecycle-of-an-offer) with each new validated ledger. This field is defined slightly differently than the version of this field in [Order Book subscription streams](subscribe.html#order-book-streams):
+
+| `Field`       | Value  | Description                                         |
+|:--------------|:-------|:----------------------------------------------------|
+| `owner_funds` | String | Numeric amount of the `TakerGets` currency that the `Account` sending this OfferCreate transaction has after the execution of all transactions in this ledger. This does not check whether the currency amount is [frozen](freezes.html). |
+
+If the request specified `"diff: true`, the response has an object `diff`. The fields of this object are as follows:
+
+| `Field`       | Value  | Description                                         |
+|:--------------|:-------|:----------------------------------------------------|
+| `object_id` | String | The object identifier. |
+| `Hashes` | Array | Array of hashes that were added, modified, or deleted as part of applying transactions in the ledger. |
```

Comment:
This should be `object`. The value is either an object, or a hex string (depending on whether the request set `binary` to true or false). The description should be something like: the contents of the object that was created, the new value of an object that was modified, or an empty string if the object was deleted.

This example output is confusing because the ledger has no transactions. We should use a ledger with some transactions as an example.